docs/src/content/ignition/docs/advanced/migrating.md

```diff
@@ -0,0 +1,382 @@
+# Migrating from hardhat-deploy
+
+For several years, the hardhat-deploy community plugin has been a go-to solution for deploying smart contracts within the Hardhat community. Recently, we introduced [Hardhat Ignition](/ignition/docs/getting-started), a declarative system for deploying smart contracts on Ethereum, which aims to pick up the torch from `hardhat-deploy` and expand Hardhat's deployment features.
+
+This guide will walk you through migrating your Hardhat project from `hardhat-deploy` to Hardhat Ignition.
```

Comment:
```suggestion
This guide will walk you through migrating your Hardhat project to Hardhat Ignition from `hardhat-deploy`, a community plugin for deploying smart contracts used within the Hardhat community.
```

I don't think my suggestion is great, but the blog intro needs reworked. What do you think?

Reply:
I think yours is fine, personally